Sources/PrimerSDK/Classes/User Interface/Primer/PaymentMethodComponent.swift

```diff
@@ -65,6 +65,16 @@ internal class PaymentMethodComponent: PrimerView {
             addSubview(iconView)
             configureIconView(icon: method.toIconName()?.image, color: theme.colorTheme.text1)
             anchorIconView(inRelationToLabel: false)
+        case .apaya:
+            layer.borderWidth = 1
+            layer.borderColor = theme.colorTheme.disabled1.cgColor
+            backgroundColor = theme.colorTheme.main1
+            label.textColor = theme.colorTheme.text1
+            addSubview(label)
+            addSubview(iconView)
+            configureLabel(with: method.toString())
+            anchorLabel()
```

Comment:
`anchorLabel` and `anchorIconView` can be unified since they're both `UIView`s and anchoring is for views. The `anchorLabel` naming is not very clear to me, since anchoring is done on another view. On which view is the label anchored?

Reply:
Will look into this and give an update. Generally, this component's logic could be greatly improved. Just not sure what's the best approach. What do you think?